### ISSUE TITLE
"Edit > To Multi Selection" visible/enabled when not useful

### ISSUE THREAD

**Issue by Bananeweizen** [eclipse-platform/eclipse.platform.ui#756]
Open an editor.
Select another view (like package explorer).

Edit>To Multi Selection is visible and enabled. That's confusing, since edit and selection commands are bound to the current selection (that's why Edit>Expand and similar are disabled when selecting a view).

Screenshot with the package explorer being the active view:
![grafik](https://github.com/eclipse-platform/eclipse.platform.ui/assets/406876/e3f646ea-0e4f-4c09-bf4a-f96e8aecf15d)

Note: The menu item is not visible, if no editor is opened. So it partially checks whether it's applicable.

**Comment by merks** [comment]
Even "To multi-selection" looks bad and should be "To Multi-selection" and even that doesn't look very nice...

**Comment by mickaelistria** [comment]
Indeed, the handler should only be enabled when current selection is a Block Text Selection. Maybe we should explicit "Convert block selection into multi-selection" ?

**Comment by Ennowulff** [comment]
What is "To Multi-Selection" even used for? what does ist mean? Thanks

**Comment by mickaelistria** [comment]
> What is "To Multi-Selection" even used for? what does ist mean? Thanks

https://eclipse.dev/eclipse/news/4.22/platform.php#multi-selection
